test/fiddle/test_fiddle.rb

```diff
@@ -27,7 +27,7 @@ def test_windows_constant
 
   def test_dlopen_linker_script_input_linux
     omit("This is only for Linux") unless RUBY_PLATFORM.match?("linux")
-    if Dir.glob("/usr/lib/*/libncurses.so").empty?
+    if Dir.glob("/usr/lib{64}/**/libncurses.so").empty?
```

Comment:
What did you intend by `{64}`?
As it doesn't contain any `,`, it matches the exact `/usr/lib{64}` directory that is usually doesn't exist.